services/response_code.proto

```diff
@@ -1116,4 +1116,19 @@ enum ResponseCodeEnum {
    * type actually has.
    */
   UNEXPECTED_TOKEN_DECIMALS = 283;
+
+  /**
+   * The proxy account id is invalid or does not exist.
+   */
+  INVALID_PROXY_ACCOUNT_ID = 284;
```

Comment:
Seems inefficient to create a new response code for every account ID field. We have hundreds of such fields. Maybe we need another field that outputs the path at which the error takes place? Perhaps consider for the future.

Reply:
As discussed, since we need specific response codes only for `auto renew account, transfer account, proxy account,  scheduling account, fee collector account`, current approach can be followed. Other places the account id being exercised is the subject that is being created/updated/deleted and can use `INVALID_ACCOUNT_ID` if it is not valid.